packages/compiler-sfc/src/compileScript.ts

```diff
@@ -1074,6 +1075,9 @@ function walkDeclaration(
     // export function foo() {} / export class Foo {}
     // export declarations must be named.
     bindings[node.id!.name] = BindingTypes.SETUP_CONST
+  } else if (node.type === 'TSEnumDeclaration' && !node.const) {
+    // treat enums as const, e.g. enum Foo { A, B }
+    bindings[node.id!.name] = BindingTypes.SETUP_CONST
```

Comment:
How about combining this branch into the one before?

Reply:
which one?

Reply:
L1077 is the same as L1080

Reply:
Oh sure, I misunderstood what you mean, 👍